2d/src/main/java/org/tweetwallfx/twod/TagTweets.java

```diff
@@ -249,26 +214,21 @@ private TweetInfo checkNewTweetHasTags(TweetInfo info) {
 
     private class TweetsUpdateTask extends Task<Void> {
```

Comment:
Evaluate: Should this be better as part of the new Tweet API abstraction layer (reduces duplication)?